app/src/main/res/layout/user_fragment_myevents.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".user.MyEventsFragment">
+
+    <TextView
+        android:id="@+id/text_home"
```

Comment:
NIT: should the naming of this be something more specific so that it is more identifiable?

Reply:
this was added missed from jens pr, since this was just a copy paste of her myevents fragment.